runtime/doc/options.txt

```diff
@@ -5385,11 +5385,34 @@ A jump table for the options with a short description can be found at |Q_op|.
 	If the name of the shell contains a space, you might need to enclose
 	it in quotes.  Example: >
 		:set shell=\"c:\program\ files\unix\sh.exe\"\ -f
-<	Note the backslash before each quote (to avoid starting a comment) and
-	each space (to avoid ending the option value).  Also note that the
-	"-f" is not inside the quotes, because it is not part of the command
-	name.  And Vim automagically recognizes the backslashes that are path
-	separators.
+<	or >
+		:let &shell='"C:\Program Files\unix\sh.exe" -f'
+<	Note the backslash before each quote in first variant (to avoid 
+	starting a comment) and each space (to avoid ending the option value) 
+	in first variant, so better use |:let-&|.  Also note that the "-f" is 
```

Comment:
The "Note the backslash ..." sentence should go directly below the first variant. Then "so better use |:let-&|" serves to introduce the `let` variant immediately below it.